vocabularies/Common.xml

```diff
@@ -1480,6 +1480,17 @@ If the request succeeds the response will contain a JSON body of `Content-Type:
         <Annotation Term="Core.Description" String="Base URL for WebSocket connections" />
       </Term>
 
+      <Term Name="Meta" Type="Common.MetaType" Nullable="false" AppliesTo="Schema">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="Meta attributes of the Schema" />
+      </Term>
+      <ComplexType Name="MetaType">
+        <Annotation Term="Common.Experimental" />
+        <Property Name="Creator" Type="Edm.String" Nullable="false">
+          <Annotation Term="Core.Description" String="Constant and version-independent identification of the schema creator" />
```

Comment:
How is uniqueness guaranteed? Should we insist that this is a URI?

Reply:
I thought a qualifier would be enough, like: 

      <Annotation Term="Common.Meta" Qualifier="CAP">
        <Record Type="Common.MetaType">
          <PropertyValue Property="Creator" String="SAP CAP Core Data Services"/>
        </Record>
      </Annotation>

Reply:
With that approach, you can have several annotations:
```xml
<Annotation Term="Common.Meta" Qualifier="CAP">
  <Record Type="Common.MetaType">
    <PropertyValue Property="Creator" String="SAP CAP Core Data Services"/>
  </Record>
</Annotation>
<Annotation Term="Common.Meta" Qualifier="RAP">
  <Record Type="Common.MetaType">
    <PropertyValue Property="Creator" String="SAP RAP Core Data Services"/>
  </Record>
</Annotation>
```
Is this intended?

Reply:
yes, if some application wants to eternalize itself, why not...

Admittedly, personally I would prefer a single attribute right in `edmx:DataServices`: low footprint, tamperproof, etc... but I think that's not easily achievable...

Reply:
@HeikoTheissen feel free to come up with a better approach, any ideas are appreciated

Reply:
Identity the creator by a URI instead of a qualifier, see https://github.com/SAP/odata-vocabularies/pull/337#discussion_r1692703125.